recipe/install-lib.sh

```diff
@@ -1,3 +1,29 @@
+mkdir -p $PREFIX/lib $PREFIX/include
+pushd cadical
+    export LDFLAGS="$LDFLAGS -Wl,-headerpad_max_install_names"
+    ./configure
+    make -j8
```

Comment:
Use $CPU_COUNT instead:
```suggestion
    make -j$CPU_COUNT
```